lib/ansible/modules/packaging/os/homebrew_cask.py

```diff
@@ -552,7 +581,23 @@ def _install_current_cask(self):
         )
 
         cmd = [opt for opt in opts if opt]
-        rc, out, err = self.module.run_command(cmd)
+
+        rc, out, err = '', '', ''
+
+        if self.sudo_password:
```

Comment:
Rather than having this same code four times in this module, turn this into a method that gets called.